admin/src/permissions.js

```diff
@@ -0,0 +1,34 @@
+const pluginPermissions = {
+  // This permission regards the main component (App) and is used to tell
+  // If the plugin link should be displayed in the menu
+  // And also if the plugin is accessible. This use case is found when a user types the url of the
+  // plugin directly in the browser
+  main: [
+    { action: 'plugins::comments.read', subject: null },
+    { action: 'plugins::comments.moderate.block.comment', subject: null },
+    { action: 'plugins::comments.moderate.block.thread', subject: null },
+    { action: 'plugins::comments.moderate.reports', subject: null },
+  ],
+  open: [
+    { action: 'plugins::comments.read', subject: null },
+  ],
+  moderate: [
+    { action: 'plugins::comments.read', subject: null },
+    { action: 'plugins::comments.moderate.block.comment', subject: null },
+    { action: 'plugins::comments.moderate.block.thread', subject: null },
+  ],
+  moderateComments: [
+    { action: 'plugins::comments.read', subject: null },
+    { action: 'plugins::comments.moderate.block.comment', subject: null },
+  ],
+  moderateThreads: [
+    { action: 'plugins::comments.read', subject: null },
+    { action: 'plugins::comments.moderate.block.thread', subject: null },
+  ],
+  moderateReports: [
+    { action: 'plugins::comments.read', subject: null },
+    { action: 'plugins::comments.moderate.reports', subject: null },
+  ],
+};
+
+export default pluginPermissions;
```

Comment:
granting these permissions is controlled by user of the plugin?
If so I think we should document them in README (with description for each permission)

Reply:
@mixvar those permissions are just definition of subsets (as strapi advice, see their internal plugins). Definition of exact permissions are in the `config/functions/bootstrap.js` file